app/views/devise/passwords/new.html.erb

```diff
@@ -1,15 +1,23 @@
-<h2>Forgot your password?</h2>
+<div class="row">
+  <div class="col-xs-8 col-sm-6 col-md-4 col-xs-offset-2 col-sm-offset-3 col-md-offset-4">
```

Comment:
Here is how this appears on an iPhone:

![image](https://cloud.githubusercontent.com/assets/7482329/26532636/63541142-43c3-11e7-8bd0-218e3b050555.png)

In order to fix this:

1. change `col-xs-8` to `col-xs-12`
2. remove `col-xs-offset-2`
3. In your [app/views/layouts/application.html.erb](https://github.com/sfillm07/proposal_app/blob/master/app/views/layouts/application.html.erb#L3-L9) file, add this inside of the `<head>` tags:

```
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

This enables you to leverage bootstrap in a [mobile first manner](http://getbootstrap.com/css/#overview-mobile).